loopchain/peer/block_manager.py

```diff
@@ -648,6 +648,10 @@ def vote_unconfirmed_block(self, block: Block, round_: int, is_validated):
             reps_hash=target_reps_hash
         )
 
+        if (self.__channel_service.inner_service.get_sub_services_properties('reject_create_tx') and
```

Comment:
The logic below doesn't work for citizen nodes.
CitizenNode also needs to be fixed so that there are no problems.